rpc/request_sign_with.go

```diff
@@ -0,0 +1,131 @@
+package rpc
```

Comment:
The file name should be something like `request_signer.go`?
I feel it's a bit weird to see a program file ending with prepositions.

Reply:
Understood.
Sorry for my bad naming.
(it was requset_signer.go in the beginning)